pvlib/par.py

```diff
@@ -0,0 +1,72 @@
+"""
+Photosynthetically Active Radiation (PAR) module.
+Utilities found here are specially interesting for agrivoltaic systems.
+"""
+
+from pvlib.tools import cosd, sind
+
+
+def spitters_relationship(solar_zenith, global_diffuse_fraction):
+    r"""
+    Derive the diffuse fraction of photosynthetically active radiation (PAR)
+    respect to the global radiation diffuse fraction.
+
+    The relationship is based on the work of Spitters et al. (1986) [1]_.
+
+    Parameters
+    ----------
+    solar_zenith : numeric
+        Solar zenith angle in degrees :math:`^{\circ}`.
+
+    global_diffuse_fraction : numeric
+        Fraction of the global radiation that is diffuse. Unitless.
+
+    Returns
+    -------
+    par_diffuse_fraction : numeric
+        Photosynthetically active radiation in W/m^2.
```

Comment:
The unit for a fraction should be 0 to 1. Otherwise. the returned parameter should be named diffsuse_par

Reply:
Quick question, is `Unitless [0, 1]` alright?

Reply:
Sounds good to me.